tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java

```diff
@@ -147,6 +157,9 @@ public void start() throws MQClientException {
 
                 mqClientInstance.start();
 
+                executors = new ThreadPoolExecutor(10, 20, 1000 * 30, TimeUnit.MILLISECONDS,
+                    new ArrayBlockingQueue<>(100), new ThreadFactoryImpl("MQAdminThread_"), new ThreadPoolExecutor.CallerRunsPolicy());
```

Comment:
I wonder why not 'CachedThreadPool', like `new ThreadPoolExecutor(0, Integer.MAX_VALUE, 1000 * 30, TimeUnit.MILLISECONDS...`.

Reply:
Good suggestion. Actually this thread pool is idle most of the time...